subprojects/docs/src/docs/release/notes.md

```diff
@@ -78,12 +82,44 @@ Scala plugin allows users to compile their Scala code using Gradle and the Zinc
 
 #### Scala 3 support
 
-Newest version of Scala 3 brings about numerous features while keeping compatibility with most of the existing 
+Newest version of Scala 3 brings about numerous features while keeping compatibility with most of the existing
 Scala 2 code. Due to some large changes in the compiler it was neccessary to adjust the Scala plugin
-to be able to compile Scala 3 code. All existing configuration options should still be usable with the newest 
-language version. To see more about the language features go to 
+to be able to compile Scala 3 code. All existing configuration options should still be usable with the newest
+language version. To see more about the language features go to
 [overview of the new features in Scala 3](https://docs.scala-lang.org/scala3/new-in-scala3.html).
 
+### Allow copying single files into directories which contain unreadable files.
+
+Sometimes you want to copy files into a directory which contains unreadable files or which is not exclusively owned by the build.
+For example when you are deploying single files into application servers or installing executables.
+This may fail or be slow since Gradle tries to track all the content in the destination directory.
+
+The `Copy` task now has a method [`Copy.ignoreExistingContentInDestinationDir()`](dsl/org.gradle.api.tasks.Copy.html#org.gradle.api.tasks.Copy:ignoreExistingContentInDestinationDir()) that forces Gradle to ignore content in the destination directory.
+This uses the new [untracked](#untracked) feature under the hood.
+See the samples in the user manual about [Deploying single files into application servers](userguide/working_with_files.html#sec:copy_deploy) and [Installing executables](userguide/working_with_files.html#sec:install_executable).
+
+<a name="plugin-development-improvements"></a>
+## Plugin development improvements
+
+<a name="untracked"></a>
+### Allow plugin authors to declare inputs or outputs as untracked
+
+For up-to-date checks and the build cache, Gradle needs to understand the inputs and outputs of a task.
+Though it is not always desirable or possible for Gradle to fully understand the input and output files.
+For example:
+- The location contains unreadable files like pipes where Gradle cannot track the content.
+- Another tool like Git already takes care of keeping the state, so it doesn't make sense for Gradle to do additional bookkeeping.
+- The build does not own the output location exclusively and Gradle would need to snapshot a potentially large amount of content.
+
+Gradle 7.3 introduces the annotation [`@Untracked`](javadoc/org/gradle/api/tasks/Untracked.html) to declare that an input or output property should not be tracked by Gradle.
```

Comment:
Shall we mention that users can also use `untracked()` in the runtime API?

Reply:
Good point. Added.